src/structures/Contact.js

```diff
@@ -120,6 +120,22 @@ class Contact extends Base {
         return await this.client.getProfilePicUrl(this.id._serialized);
     }
 
+    /**
+     * Returns the contact's formatted phone number, (+1 (234) 5678-901)
+     * @returns {Promise<string>}
+     */
+    async getFormattedId() {
```

Comment:
`getFormattedId` might be misleading since this doesnt quite represent a contact id. Maybe a better name is `getFormattedNumber`

Reply:
Agreed